projects/plugins/jetpack/modules/masterbar/admin-menu/class-base-admin-menu.php

```diff
@@ -389,16 +389,25 @@ public function override_svg_icons() {
 
 		$svg_items = array();
 		foreach ( $menu as $idx => $menu_item ) {
-			if ( count( $menu_item ) > 6 && 0 === strpos( $menu_item[6], 'data:image/svg+xml' ) && 'site-card' !== $menu_item[3] ) {
-				$svg_items[] = array(
-					'icon' => $menu[ $idx ][6],
-					'id'   => $menu[ $idx ][5],
+			if ( count( $menu_item ) < 7 ) {
+				continue;
+			}
+
+			// If the hookname looks to contain a URL than sanitize it by remplacing invalid characters.
```

Comment:
Should this be "If the hookname contains a URL then sanitize it by replacing invalid characters"?

Reply:
Yes, that's better! I think it was because the check is for `://` so it's a URL-like string, rather than being sure we've got a URL. Your version is simpler though